internal/state/state.go

```diff
@@ -240,13 +240,21 @@ func (state State) MakeBlock(
 	// Build base block with block data.
 	block := types.MakeBlock(height, txs, lastCommit, evidence)
 
-	// TODO(@glnro): Update logic once merged with #2205
-	// Check if PBTS Enabled: state.ConsensusParams.PBTS.PBTSEnabled(height)
+	// Set time.
+	var timestamp time.Time
+	switch {
+	case state.ConsensusParams.Feature.PbtsEnabled(height):
+		timestamp = time.Now()
```

Comment:
Why `time` and not `cmttime` here?

Reply:
Notice that they are different:

```
=== RUN   TestTime
    time_test.go:11: cmttime.Now() 2024-02-15 16:47:32.96113 +0000 UTC
    time_test.go:12: time.Now() 2024-02-15 17:47:32.961536 +0100 CET m=+0.012614460
```

Reply:
The question here is how that hell this doesn't break any test.

Reply:
Internally they are the same; `cmttime` is just `time` at UTC, but the localtime is the same. In any case, changing to `cmttime`.

Reply:
It is also stripped.